src/main/ngapp/accio/src/app/img-gallery/img-gallery.component.ts

```diff
@@ -1,16 +1,115 @@
 import { Component, OnInit } from '@angular/core';
-import { imageUrls } from '../images'
+import { ActivatedRoute } from '@angular/router';
+import { FormGroup, FormControl } from '@angular/forms';
+import { PostBlobsService } from '../post-blobs.service';
+import * as $ from 'jquery';
 
 @Component({
   selector: 'app-img-gallery',
   templateUrl: './img-gallery.component.html',
   styleUrls: ['./img-gallery.component.css']
 })
 export class ImgGalleryComponent implements OnInit {
-  imageUrls = imageUrls;
-  constructor() { }
+  uploadImageForm: FormGroup;
+  formData: FormData;
+  mode: string = 'create';
+  projectId: string;
+
+  displayUpload: boolean = false;
+  displayImages: boolean = false;
+
+  // Holds images fetched from datastore
+  imageArray: Array<any>;
+
+  constructor(
+    private route: ActivatedRoute,
+    private postBlobsService: PostBlobsService,
+  ) { }
 
   ngOnInit(): void {
+    this.uploadImageForm = new FormGroup({
+      imgName: new FormControl(),
+      image: new FormControl(),
+      tags: new FormControl()
+    });
+
+    //creates the form data of parameters to be sent to servlet
```

Comment:
nit: consistency in comments; sometimes there is a space after the // and sometimes not, also sometimes there is capitalization on the first char and sometimes not.